docs/FrontierDevelopments-CAPI-endpoints.md

```diff
@@ -14,138 +16,373 @@ It's safest to store the token_type that comes back from the request for
+		1. `visitDurationSeconds` - How many seconds were spent in the system
+		1. `starsystem` - The visited starsystem
+	1. `totalDistanceJumpedLY` - How many lightyears this carrier has ever jumped
+	1. `currentJump` - The currently scheduled jump, can be null
+1. `marketFinance`
+	1. `cargoTotalValue` - The total value (credits) of the cargo
+	1. `allTimeProfit` - How much profit this carrier has made, all time
+	1. `numCommodsForSale` - How many commodities are for sale
+	1. `numCommodsPurchaseOrders` - How many purchase orders for commodities there are
+	1. `balanceAllocForPurchaseOrders` - How many credits are allocated for purchase orders
+1. `blackmarketFinance`
+	1. `cargoTotalValue` - The total value (credits) of the cargo
+	1. `allTimeProfit` - How much profit this carrier has made, all time
+	1. `numCommodsForSale` - How many commodities are for sale
+	1. `numCommodsPurchaseOrders` - How many purchase orders for commodities there are
+	1. `balanceAllocForPurchaseOrders` - How many credits are allocated for purchase orders
+1. `finance`
+	1. `bankBalance` - How much credits are in the bank of the carrier
+	1. `bankReservedBalance` - How much of said credits are reserved
+	1. `taxation` - The taxation on sales/purchases
+	1. `numServices` - How many services are active
+	1. `numOptionalServices` - How many optional services are active
+	1. `debtThreshold` - Declares the debt threshold
+	1. `maintenance` - How much maintenance costs
+	1. `maintenanceToDate` - How much maintenance has been paid to date
+	1. `coreCost` - Core costs for the carrier
+	1. `servicesCost` - The costs for services
+	1. `servicesCostToDate` - How much services costs has been paid to date
+	1. `jumpsCost` - How much a jump costs
+	1. `numJumps` - Number of jumps
+1. `servicesCrew`
+1. `cargo`
+	1. `commodity` - The commodity available
+	1. `mission` - Is this a mission commodity?
+	1. `qty` - The quantity available of this commodity
+	1. `value` - The value (credits) of this commodity
+	1. `stolen` - Was this item stolen?
+	1. `locName` - Localised name of the commodity
+1. `reputation`
+	- Reputation is an array with objects that describe ties to the major factions
+	1. `majorFaction` - The major faction
+	1. `score` - The reputation score
+1. `market`
+	- Contains the same information as a call to [`/market`](#market), check that for more info
+1. `ships`
+	- Contains the same information as the modules in [`/shipyard`](#shipyard), check that for more info
+1. `modules`
+	- Contains the same information as the modules in [`/shipyard`](#shipyard), check that for more info
+
+&nbsp;
+
+---
+
 ## Market
+
+    	GET <https://companion.orerve.net/market>
+
+Get the market data from the last docked station/fleet carrier/settlement and contains their services and status, what type of economics it has, what they import, export and what is forbidden. (And also the commodities, with price info and stock/demand)
+
```

Comment:
Maybe add a reference to the FDevIDs repo as a place to get the known commodity IDs ?